appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -272,6 +280,12 @@ static ControllerLinkBuilder getAddUserLinkBuilder()
 		return linkTo(methodOn.addUser(Optional.empty(), null, null, null));
 	}
 
+	static ControllerLinkBuilder getUpdateUserLinkBuilder(UUID userId)
+	{
+		UserController methodOn = methodOn(UserController.class);
+		return linkTo(methodOn.updateUser(Optional.empty(), null, userId, null, null));
```

Comment:
This effectively makes it impossible for a user to update their account more than x times in the defined time window. That sounds quite reasonable to me.

Looks like we should make the window rather large (e.g. an hour) and the number of attempts relatively high (10 times). That way, normal usage will never hit this, but someone probing numbers will only be able to do 10 per hour.

For {{addUser}}, the scenario is different, as that is global, for all users. If we would use the same window and limit, then we could only add 10 users per hour, which is too low. In that case, it might make more sense to make the window shorter, so we hinder people that probe numbers but we do not block onboarding of considerable numbers of users (e.g. in the break of a Yona promotion meeting). If we make the window shorter (e.g. 1 minute) and the limit low (e.g. 1), we can add a user per minute, which is quite OK, but someone probing numbers can only try 60 per hour.

Makes sense?